client/src/components/chess-game/ChessGame.js

```diff
@@ -118,7 +125,7 @@ export default class ChessGame extends React.Component {
     }
   };
 
-  startGame() {
+  startGame = () => {
```

Comment:
:+1:

Reply:
Make sure we don't have dangling `bind` statements in constructor, as this syntax removes it's usage.